config.py

```diff
@@ -105,3 +105,18 @@ def user_configuration():
         "default_selected_modules": "all",  # or select one or multiple (e.g. ftp/strong_password,ssh/strong_password)
         "default_excluded_modules": None  # or any module name separated with comma
     }
+
+
+def sentry_configuration() -> dict:
+    """
+        sentry configuration
+
+    Returns:
+        JSON/Dict sentry configuration
```

Comment:
this would need to be documented in the wiki too I believe after merging

Reply:
yes, will add proper documentation in wiki